### PR TITLE
fix go get sub package and add domain on installation to let go get work defaultly

### LINKED ISSUE #1485: Make go packages from subdirectories on gitea instances installable with "go get"
Currently, you can't fetch go projects from gitea, without the .git extension. This causes the binaries to have this ugly .git extension too and prevents people from fetching projects from subdirectories.

Documentation of meta-tags: https://golang.org/cmd/go/#hdr-Remote_import_paths
The PR and commit that fixed this in gitlab: https://github.com/gitlabhq/gitlabhq/pull/7693 https://github.com/gitlabhq/gitlabhq/commit/e23d6ada1c234acd45b676a34a2997aeae94d087

### PR DESCRIPTION
**Pull request by lunny**
fix #1485

### CODE REVIEW

File: routers/repo/http.go [hunk 1]
```diff
@@ -17,18 +17,42 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Unknwon/com"
```
Comment (cez81) [inline routers/repo/http.go]: External after local imports?

Comment (lunny) [reply routers/repo/http.go]: good catch and done.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by strk** [comment]
LGTM but it will look much better if it came with unit tests.
@lunny, as an owner, you should be an example to others ! :)

**Comment by lunny** [comment]
@strk in fact, the unit tests depends on integration test framework. I think I will do that after that's ready.

**Comment by lunny** [comment]
let L-G-T-M work

**Pull request merged**